docs/using/libraries/java/index.rst

```diff
@@ -212,18 +217,18 @@ Likewise, dictionaries constructed in Polar may be passed into Java methods.
 
 Enumerations
 ^^^^^^^^^^^^
-Oso handles Java objects that implement the `Enumeration <https://docs.oracle.com/javase/10/docs/api/java/util/Enumeration.html>`_ interface by evaluating each of the
-object's elements one at a time:
+You may iterate over a Java `Enumeration <https://docs.oracle.com/javase/10/docs/api/java/util/Enumeration.html>`_
+using the Polar :ref:`operator-in` operator:
 
 .. code-block:: polar
    :caption: :fa:`oso` policy.polar
 
-   allow(actor, action, resource) if actor.getGroup = "payroll";
```

Comment:
Ah crap, good catch! My regex failed me since I was only looking for `<blah> = <x>.<y>`